core/nodepool/config/templates/stack-template.json

```diff
@@ -12,7 +12,10 @@
   "   --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf  \\",
   "   --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos \\",
   "   --trust-keys-from-https \\",
-  "   {{.AWSCliImageRepo}}:{{.AWSCliTag}} -- aws s3 --region $REGION  cp {{ .S3URI }}/{{ .StackName }}/$USERDATA_FILE /var/run/coreos/",
+  {{- if .AWSCliImage.RktPullDocker }}
+  "   --insecure-options=image \\",
+  {{- end }}
```

Comment:
To reduce `if`s in templates and make it more testable, how about implementing something like:

```golang
func (i Image) RktOptions() string {
    if i.RktPullDocker {
        return "--insecure-options=image"
    }
    return ""
}
```

and refer to it from here?